components/proxy/Dockerfile

```diff
@@ -18,7 +18,8 @@ RUN xcaddy build v2.6.2 \
   --with github.com/gitpod-io/gitpod/proxy/plugins/configcat=/plugins/configcat \
   --with github.com/gitpod-io/gitpod/proxy/plugins/logif=/plugins/logif \
   --with github.com/gitpod-io/gitpod/proxy/plugins/jsonselect=/plugins/jsonselect \
-  --with github.com/gitpod-io/gitpod/proxy/plugins/sshtunnel=/plugins/sshtunnel
+  --with github.com/gitpod-io/gitpod/proxy/plugins/sshtunnel=/plugins/sshtunnel \
+  --with github.com/gitpod-io/gitpod/proxy/plugins/fe_dev=/plugins/fe_dev
```

Comment:
in the context of this PR I assume fe = frontend, but when I stumble over this name in a few weeks I'll have no idea.

Reply:
Jup, don't like that name either. Also, we should find a better name for `X-FE-DEV-URL` :laughing: 

Maybe:
- `X-DEV-URL`
- ...

Reply:
I would go with `X-Frontend-Dev-URL` for the header and `frontend_dev` for the plugin name.

Reply:
Done.